src/main/java/com/epam/task/first/logic/ArrayLogic.java

```diff
@@ -0,0 +1,112 @@
+package com.epam.task.first.logic;
+
+import com.epam.task.first.entity.Array;
+
+import java.util.ArrayList;
+
+public class ArrayLogic {
+    private Array array;
+
+    public ArrayLogic(Array array) {
+        this.array = array;
+    }
+
+    public int findMax() {
+        ArrayList<Integer> elements = array.getElements();
+        if (elements.size() == 0) {
```

Comment:
I would say you can omit this check for now, don't use runtime exception now as well